docs/outlook/troubleshoot-outlook-add-in-activation.md

```diff
@@ -227,15 +227,15 @@ If the **ItemHasRegularExpressionMatch** activation rule specifies **Subject** o
 
 After verifying the property value, you can then use a regular expression evaluation tool to test whether the regular expression finds a match in that value.
 
-## Does the host application apply all the regular expressions to the portion of the item body as you expect?
+## Does the Outlook application apply all the regular expressions to the portion of the item body as you expect?
 
-This section applies to all activation rules that use regular expressions -- particularly those that are applied to the item body, which may be large in size and take longer to evaluate for matches. You should be aware that even if the item property that an activation rule depends on has the value you expect, the host application may not be able to evaluate all the regular expressions on the entire value of the item property. To provide reasonable performance and to control excessive resource usage by a read add-in, Outlook, Outlook on the web and mobile devices observe the following limits on processing regular expressions in activation rules at run time:
+This section applies to all activation rules that use regular expressions -- particularly those that are applied to the item body, which may be large in size and take longer to evaluate for matches. You should be aware that even if the item property that an activation rule depends on has the value you expect, the Outlook application may not be able to evaluate all the regular expressions on the entire value of the item property. To provide reasonable performance and to control excessive resource usage by a read add-in, Outlook, Outlook on the web and mobile devices observe the following limits on processing regular expressions in activation rules at run time:
```

Comment:
Suggesting to replace "the Outlook application" and "Outlook, Outlook on the web and mobile devices" with Outlook
```suggestion
This section applies to all activation rules that use regular expressions -- particularly those that are applied to the item body, which may be large in size and take longer to evaluate for matches. You should be aware that even if the item property that an activation rule depends on has the value you expect, Outlook may not be able to evaluate all the regular expressions on the entire value of the item property. To provide reasonable performance and to control excessive resource usage by a read add-in, Outlook observes the following limits on processing regular expressions in activation rules at run time:
```